phys/module_mp_thompson.F

```diff
@@ -419,7 +419,7 @@ SUBROUTINE thompson_init(hgt, orho, nwfa2d, nwfa, nifa,           &
 
 
       INTEGER:: i, j, k, l, m, n
-      REAL:: h_01, airmass, niIN3, niCCN3, max_test
+      REAL:: h_01, airmass, niIN3, niCCN3, max_test, z1 !mchen
```

Comment:
Ming you can take your name off this line

Reply:
Jimy,
It has been changed.